cfs-require-import.php

```diff
@@ -8,32 +8,50 @@
 Author URI: https://www.vanpattenmedia.com/
 */
 
-class CfsRequireImport {
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```

Comment:
I'm okay with including this but also wonder if the solution might be to do the `is_plugin_active` check inside an `is_admin` check, so it can only try to run the import if you're logged inside wp-admin. Then you wouldn't need this include. Curious what you think!

Reply:
It requires `add_action( 'admin_init'...` to not include.php when using `is_plugin_active`. You solution with `is_admin` is great and I found simple and elegant code at woocommerce https://docs.woocommerce.com/document/create-a-plugin/ - this is almost the same as `is_plugin_active` and I have combined both:
```
if ( is_admin() && in_array( 'custom-field-suite/cfs.php', apply_filters( 'active_plugins', get_option( 'active_plugins' ) ) ) ) {
    new CfsRequireImport();
}
```
I have tested it and it works fine. What do you think about it?